### PR TITLE
(DOCS-15355) Fix example

### PR DESCRIPTION
**Pull request by zach-carr**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-spark-connector/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCS-15355
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/spark-connector/docsworker-xlarge/DOCS-15355/structured-streaming/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

**Review by rozza — APPROVED** [review]
LGTM - good catch on the csv

File: source/includes/warn-console-stream.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.. important::
+
+   Streaming to your console is memory intensive and intended 
+   only for testing purposes. Avoid streaming large datasets 
+   to your console.
```
Comment (kyuan-mongodb) [inline source/includes/warn-console-stream.rst:5]: Suggestion: Reverse the order of the sentences. I think explaining the "why" factor first is a bit confusing if the "what" factor isn't presented first.

**Review by kyuan-mongodb — APPROVED** [review]
LGTM % a suggestion!

File: source/includes/warn-console-stream.rst (hunk 1 above)
Comment (zach-carr) [reply source/includes/warn-console-stream.rst:5]: Good call

**Pull request merged**